service/src/main/kotlin/io/provenance/digitalcurrency/consortium/service/PbcService.kt

```diff
@@ -59,8 +59,8 @@ class PbcService(
     ) {
         TESTNET_HARDENED("tp", "m/44'/1'/0'/0/0'"),
         TESTNET("tp", "m/44'/1'/0'/0/0"),
-        MAINNET_HARDENED("pb", "m/505'/1'/0'/0/0'"),
-        MAINNET("pb", "m/505'/1'/0'/0/0")
+        MAINNET_HARDENED("pb", "m/44'/505'/0'/0/0'"),
+        MAINNET("pb", "m/44'/505'/0'/0/0")
```

Comment:
wait where did these come from? we're already using the ones being removed

Reply:
how could they be correct? when I tried to use them in the old omnibus it failed until I fixed it. 

```
All interactions with Provenance Blockchain are secured with a public/private key pair that will act as your account(s) on the blockchain. We use the 44'/1'/0'/0/0 BIP32 path as an example where the coin-type "1" is a reference to Provenance Blockchain testnet. We use the 44'/505'/0'/0/0 BIP32 path as an example where coin-type "505" is a reference to the HASH token on Provenance Blockchain mainnet.
```

Reply:
well this will break nycb - also this was what was set in https://github.com/provenance-io/pb-grpc-client-kotlin/pull/28/files#diff-3ca603365491e112e42bcfd950e45bfed02b499f8bd09e843d0f275c5c81b9f7L17

at this point there are so many contradictory derivation paths i dont know what the truth is anymore

Reply:
devil's advocate but are we sure nycb is in prod with this version? We decided to pull those out of the common lib because we needed additional support for the hardened path in prod that wasn't there - which you got around by ^^. By the time I saw the discrepancy between the value and the docs the PR was merged. I don't know how we told NYCB to create their prod key but I feel like if it came from the docs it would be using `"m/44'/505'/0'/0/0"` and not `"m/505'/1'0'/0/0"`. If they are in prod with this version and it is working, we will have to add that as a backwards compatible support for them because I would think future dccs clients will create keys using the doc instructions.

Reply:
I suspect we would have told them `"m/44'/505'/0'/0/0"` or `"m/44'/505'/0'/0/0'"` but I can't remember.

Reply:
hmmm i think they used default provenanced derivation path for generating keys on mainnet, i can double check

they're probably not on this version on prod since they're on 0.13.2 still

Reply:
so I advocate for pushing this through because I think that what was in the common lib was completely wrong for mainnet.

Reply:
confirmed your changes are right 🙇